components/content_settings/core/browser/brave_content_settings_utils.cc

```diff
@@ -21,6 +23,32 @@ const std::vector<std::string> kShieldsResourceIDs {
     brave_shields::kReferrers,
     brave_shields::kCookies };
 
+bool CanPatternBeConvertedToWildcardSchemeAndPort(
+    const ContentSettingsPattern& pattern) {
+  // 1. Wildcard is alerady in the desired state.
```

Comment:
typo

Reply:
Fixed in bdc2851b5386b1ae42a134550a217edb6267ef57